app/controllers/profiles_controller.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 # Controller for the user profile page
-class ProfilesController < ApplicationController
+class ProfilesController < Profiles::ApplicationController
   layout 'profiles'
```

Comment:
Since you are basing this off of Profiles::ApplicationController you can remove the layout line.

Reply:
Updated in 6dc4a9d5114f8c5f8ef460d3b5108c373769273e